tests/components/grid.test.ts

```diff
@@ -1322,16 +1305,7 @@ describe("Events on Grid update viewport correctly", () => {
 
   test("Move selection horizontally (right to left) through pane division does not reset the scroll", async () => {
     freezeColumns(model, 3);
-    document.activeElement!.dispatchEvent(
-      // scroll completely to the right
-      new WheelEvent("wheel", {
-        deltaX: 0,
-        deltaY: 4 * DEFAULT_CELL_WIDTH,
-        shiftKey: true,
-        deltaMode: 0,
-        bubbles: true,
-      })
-    );
+    triggerWheelEvent(".o-grid input", { deltaY: 4 * DEFAULT_CELL_WIDTH, shiftKey: true });
```

Comment:
`triggerWheelEvent(document.activeElement!, ...`
to keep the same test as before